bayesian/api_v1.py

```diff
@@ -679,6 +692,7 @@ def post():
 add_resource_no_matter_slashes(StackAnalysesGET, '/stack-analyses/<external_request_id>')
 add_resource_no_matter_slashes(UserFeedback, '/user-feedback')
 add_resource_no_matter_slashes(UserIntent, '/user-intent')
+add_resource_no_matter_slashes(UserIntentGet, '/user-intent/<user>/<ecosystem>')
```

Comment:
What about adjusting only the line above? That should do the trick. Everything else can stay untouched.

Reply:
how do you intent to do it ?

Reply:
See https://github.com/fabric8-analytics/fabric8-analytics-server/pull/186 if that would work for you.

Reply:
if i have a single end point , how will my post request get served ? @fridex

Reply:
Agh, good point, I missed the POST endpoint. PR has been adjusted, please take a look.